src/protocols/secure_channel/SessionEstablishmentDelegate.h

```diff
@@ -25,13 +25,27 @@
 
 #pragma once
 
+#include <protocols/secure_channel/PairingSession.h>
 #include <system/SystemPacketBuffer.h>
 #include <transport/Session.h>
 #include <transport/raw/MessageHeader.h>
 #include <transport/raw/PeerAddress.h>
 
 namespace chip {
 
+enum class SessionState : uint8_t
```

Comment:
So I think this would be clearer as SessionEstablishmentStage.  Then we can add stages here that have to do with address lookup and whatnot later, right?